hmda/src/main/scala/hmda/validation/rules/lar/quality/Q632_2019.scala

```diff
@@ -0,0 +1,38 @@
+package hmda.validation.rules.lar.quality.LAR
```

Comment:
For the validity edits we've been using the convention of breaking the edits down into `common`, `2018`, and `2019` folders and then adding `eighteen` and `nineteen` to the package to differentiate the classes. I think that we should stick with the same convention for the quality edits.